src/utils/uint64.ts

```diff
@@ -0,0 +1,18 @@
+// It's called Int64, but it's a UInt64
```

Comment:
What it says. The library is called Int64, but it's actually implementing an unsigned integer.